src/actions/biblioActions.js

```diff
@@ -438,58 +435,47 @@ export const wb_entity_validation = (dispatch, entityType, entityInputList, call
 };
 
 
-async function callATeamApi(ateamApiUrl, postData, accessToken) {
-  try {
-    const res = await axios.post(ateamApiUrl, postData, {
-      headers: {
-        'content-type': 'application/json',
-        'authorization': 'Bearer ' + accessToken
-      }
-    });
-    return res.data.results || [];
-  } catch (err) {
-    console.error("Error in entity lookup API:", err);
-    throw err;
-  }
-}
-
+export const abc_entity_validation = (dispatch, entityType, entityInputList, taxon, callback) => {
 
-function processApiResults(results, taxon, entityType) {
-  const searchMap = {};
-  const obsoleteMap = {};
-  for (const entityResult of results) {
-    if (['gene', 'allele'].includes(entityType) && entityResult.taxon.curie !== taxon) {
-      continue;
-    }
-    let primaryId = entityResult.curie ? entityResult.curie : entityResult.modEntityId;
-    let name = entityResult.name ? entityResult.name.toLowerCase() : entityResult[entityType + 'Symbol'].displayText.toLowerCase();
-    let systematicName = entityType === "gene" ? entityResult['geneSystematicName']?.displayText?.toLowerCase() ?? "" : "";
-    let otherName = entityType === "construct" ? entityResult['constructFullName']?.displayText?.toLowerCase() ?? "" : "";
-    if (primaryId && name) {
-      if (entityResult.obsolete === true) {
-        obsoleteMap[primaryId.toLowerCase()] = primaryId;
-        obsoleteMap[name] = primaryId;
-        if (systematicName) {
-          obsoleteMap[systematicName] = primaryId;
+    const entityListStr = entityInputList.join('|');
+    const encodedEntityList = encodeURIComponent(entityListStr);
+    const url = `${restUrl}/topic_entity_tag/entity_validation/${taxon}/${entityType}/${encodedEntityList}`;
```

Comment:
Is this GET a potential limit on the number of entities we can pass to the API? If so we could make it a POST and modify the API

Reply:
Good point. However, this is only for entity validation. I’m assuming curators won’t post a huge list to the entity box, so using the GET method should be sufficient. What do you think?

Reply:
Oh, good point, so the mapping for TETs is done in the backend directly, right? Then this should be fine. I don't know how many entities curators can enter in the autocomplete